### PR TITLE
Stop uploading split test coverage.

### PR DESCRIPTION
**Pull request by s0undt3ch**
This will easy the codecov service which we've been hammering hard
according to them.

### CODE REVIEW

**Comment by s0undt3ch** [comment]
Test run https://github.com/saltstack/salt/pull/55543

**Comment by s0undt3ch** [comment]
More test runs of these changes https://github.com/saltstack/salt/pull/55043

File: vars/runTestSuite.groovy [hunk 1]
```diff
@@ -34,10 +44,10 @@ def call(Map options) {
     def String kitchen_platforms_file = options.get('kitchen_platforms_file', '/var/jenkins/workspace/nox-platforms.yml')
     def String[] extra_codecov_flags = options.get('extra_codecov_flags', [])
     def String ami_image_id = options.get('ami_image_id', '')
-    def Boolean retrying = options.get('retrying', false)
     def Boolean upload_test_coverage = options.get('upload_test_coverage', true)
+    def Boolean upload_split_test_coverage = options.get('upload_split_test_coverage', false)
     def Integer concurrent_builds = options.get('concurrent_builds', 1)
-    def String test_suite_name = options.get('test_suite_name', null)
```
Comment (bryceml) [inline vars/runTestSuite.groovy]: what does test_suite_name mean?  it's not very obvious.  I would either rename it or comment above it explaining what it does.

Comment (s0undt3ch) [reply vars/runTestSuite.groovy]: That's code that was "imported" from the split test suite changes.

Once those are in, this will make more sense.

**Pull request merged**